packages/build/src/test/service-worker_test.ts

```diff
@@ -174,56 +161,19 @@ suite('service-worker', () => {
                 assert.include(fileContents, '"bower_components/dep.html"');
                 assert.notInclude(fileContents, '"source-dir/my-app.html"');
               });
-        });
-
-    test(
-        'should add bundled precached assets when options.bundled is provided',
```

Comment:
We still want to make sure that we precache bundles, yes?

Reply:
Added back